llvm/lib/CodeGen/AsmPrinter/DwarfDebug.h

```diff
@@ -107,6 +107,9 @@ class DbgVariable;
 struct FrameIndexExpr {
   int FI;
   const DIExpression *Expr;
+
+  /// Operator enabling sorting based on fragment offset.
+  bool operator<(const FrameIndexExpr &Other) const;
```

Comment:
Usually it's preferable to implement any operator overload that can be a non-member, as a non-member (to allow equal conversions on both the LHS and RHS) - could you do that here? (& elsewhere)

Reply:
Sounds good! I rolled the change to the other type here into this PR as it seems trivial and the PR process in GH doesn't let me "stack" it easily. Let me know if anyone prefers it go in separately!